libraries/AP_AccelCal/AP_AccelCal.cpp

```diff
@@ -124,6 +124,7 @@ void AP_AccelCal::update()
                 return;
             case ACCEL_CAL_SUCCESS:
                 // save
+                _last_result = ACCEL_CAL_SUCCESS;
```

Comment:
Why isn't this done in the success method? I think the __acal_get_fail_ actually doesn't do anything in any client, but in the possibility it changes, it is better to change it to the success method - and that is also where actually the success happens.